src/internal/m365/collection/drive/delta_tree.go

```diff
@@ -66,7 +67,7 @@ func newFolderyMcFolderFace(
 
 // reset erases all data contained in the tree.  This is intended for
 // tracking a delta enumeration reset, not for tree re-use, and will
-// cause the tree to flag itself as dirty in order to appropriately
+// cause the tree to flag itfolder as dirty in order to appropriately
```

Comment:
🤔 not sure how this one popped out, but I think it's safe to revert this change 😉

Reply:
no one expects the find&replace!